src/modules/Users.pm

```diff
@@ -3736,7 +3436,6 @@ BEGIN { $TYPEINFO{SubstituteGroupValues} = ["function", "void"] }
 sub SubstituteGroupValues {
 
     my $self	= shift;
-    my $substituted = UsersLDAP->SubstituteValues ("group", \%group_in_work);
```

Comment:
Here the declaration is removed, but not the use of $substituted in the following lines.

Reply:
and same here